### PR TITLE
Transaction by type + #accounts with CCD transfers

### PR DESCRIPTION
**Pull request by lassemoldrup**
## Purpose

Addresses #89 and #90.

## Changes

The information was already in the database, so--aside from an added function in `schema.sql`--only Grafana changes were needed.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: kpi-tracker/grafana/dashboard.json [hunk 1]
```diff
@@ -652,6 +657,116 @@
...
+        "tooltip": {
+          "mode": "single",
+          "sort": "none"
+        }
+      },
+      "targets": [
+        {
+          "datasource": {
+            "type": "postgres",
+            "uid": "d4ab35f1-5c8e-48b6-8566-37f813b41e94"
```
Comment (soerenbf) [inline kpi-tracker/grafana/dashboard.json:741]: This is needed to align the datasource specified elsewhere.

```suggestion
            "uid": "${postgres_db}"
```

**Review by soerenbf — APPROVED** [review]
1 small thing, otherwise looks good.

File: kpi-tracker/grafana/dashboard.json [hunk 2]
```diff
@@ -1321,6 +1444,6 @@
   "timezone": "",
   "title": "KPI tracker",
   "uid": "8dm2sVA4k",
-  "version": 35,
+  "version": 10,
```
Comment (abizjak) [inline kpi-tracker/grafana/dashboard.json:1447]: This is an odd change. What is it for?

**Review by abizjak — APPROVED** [review]
Assuming you address @soerenbf 's comment about parametrizing the data source I think this is fine.

File: kpi-tracker/grafana/dashboard.json (hunk 2 above)
Comment (lassemoldrup) [reply kpi-tracker/grafana/dashboard.json:1447]: This is just something that Grafana does automatically

**Pull request merged**